src/werkzeug/wrappers/base_request.py

```diff
@@ -615,23 +613,21 @@ def host(self) -> str:
         """
         return get_host(self.environ, trusted_hosts=self.trusted_hosts)
 
-    query_string = environ_property(
+    query_string = environ_property[bytes](
         "QUERY_STRING",
-        "",
-        read_only=True,
+        b"",
         load_func=lambda x: x.encode("latin1"),
         doc="The URL parameters as raw bytes.",
     )
     method = environ_property(
         "REQUEST_METHOD",
         "GET",
-        read_only=True,
```

Comment:
These meant to be deleted? (read_only)

Reply:
Good catch, I had noticed this too but forgot to follow up. I'll remove them.

Reply:
Misinterpreted your comment. `environ_property` defaults to `read_only=True`, so these weren't needed.